### ISSUE TITLE
Wrong #elements in prelab2.ipynb

### ISSUE THREAD

**Issue by Aratz** [scicompuu/sesegpu#2]
In the second cell, variable `elems` and the sum later in the loop should take their data from the same array. Right now `elems` reads the shape from the training set while the sum uses the test set.
